fiftyone/plugins/core.py

```diff
@@ -490,7 +492,10 @@ def _parse_plugin_metadata(metadata_path):
 
 def _list_plugins(enabled=None):
     plugins = []
-    for metadata_path in _iter_plugin_metadata_files():
+    external_plugin_paths = _iter_plugin_metadata_files()
+    builtin_plugin_paths = _iter_plugin_metadata_files(BUILTIN_PLUGINS_DIR)
```

Comment:
This now includes built ins in all listings. Lets discuss how we actually want to handle this. More thought needed here.

Reply:
I'm leaning towards something like `include_builtins=False`.